### PR TITLE
Ananas test

### PR DESCRIPTION
**Pull request by simmi-deligence**

### CODE REVIEW

File: rest_api/models/product_pricelist.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+from itertools import chain
+
+from odoo import api, fields, models, tools, _
+from odoo.exceptions import UserError, ValidationError
+
+
+class ChangePricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    def get_product_price(self, product, quantity, partner, date=False, uom_id=False):
```
Comment (bakboka) [inline rest_api/models/product_pricelist.py:14]: Why overwrite this function is it is the same as standard odoo?

File: ananas_custom_product/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
```
Comment (bakboka) [inline ananas_custom_product/__manifest__.py:1]: Please remove this module as everything is commented out

File: rest_api/models/product_pricelist.py [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+        """ For a given pricelist, return price for a given product """
+        self.ensure_one()
+        return self._compute_price_rule([(product, quantity, partner)], date=date, uom_id=uom_id)[product.id][0]
+
+    def _compute_price_rule(self, products_qty_partner, date=False, uom_id=False):
```
Comment (bakboka) [inline rest_api/models/product_pricelist.py:19]: Same thing as before, I do not see what change is being made from the standard function

File: rest_api/models/product_pricelist.py [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+
+            if not suitable_rule:
+                cur = product.currency_id
+                price = cur._convert(price, self.currency_id, self.env.company, date, round=False)
+
+            results[product.id] = (price, suitable_rule and suitable_rule.id or False)
+
+        return results
+
+    def _compute_price_rule_get_items(self, products_qty_partner, date, uom_id, prod_tmpl_ids, prod_ids, categ_ids):
```
Comment (bakboka) [inline rest_api/models/product_pricelist.py:164]: This is the same as standard function. please remove it

**Review by bakboka — CHANGES_REQUESTED** [review]

File: ananas_custom_product/__manifest__.py (hunk 2 above)
Comment (simmi-deligence) [reply ananas_custom_product/__manifest__.py:1]: ok will remove it.

File: rest_api/models/product_pricelist.py (hunk 1 above)
Comment (simmi-deligence) [reply rest_api/models/product_pricelist.py:14]: will remove it.

File: rest_api/models/product_pricelist.py (hunk 3 above)
Comment (simmi-deligence) [reply rest_api/models/product_pricelist.py:19]: will remove it.

File: rest_api/models/product_pricelist.py (hunk 4 above)
Comment (simmi-deligence) [reply rest_api/models/product_pricelist.py:164]: There in query you can see the different field for start and end date from standard odoo function.

File: rest_api/models/product_pricelist.py (hunk 3 above)
Comment (simmi-deligence) [reply rest_api/models/product_pricelist.py:19]: I have changed the date format here from date to dateTime.

**Pull request closed without merging**